tests/v1/test_parameter_mapping.py

```diff
@@ -364,15 +358,14 @@ def test_partial_override(condition_df_2_conditions):
         )
         condition_df.set_index("conditionId", inplace=True)
 
-        import simplesbml
-
-        ss_model = simplesbml.SbmlModel()
-        ss_model.addParameter("fixedParameter1", 0.5)
-        ss_model.addParameter("fixedParameter2", 1.0)
-        ss_model.addParameter("dynamicParameter1", 0.0)
-        ss_model.addParameter("observableParameter1_obs1", 0.0)
-        ss_model.addParameter("observableParameter2_obs1", 0.0)
-        ss_model.addParameter("observableParameter1_obs2", 0.0)
+        model = SbmlModel.from_antimony(
+            "fixedParameter1 = 0.5\n"
+            "fixedParameter2 = 1.0\n"
+            "dynamicParameter1 = 0.0\n"
+            "observableParameter1_obs1 = 0.0\n"
+            "observableParameter2_obs1 = 0.0\n"
+            "observableParameter1_obs2 = 0.0"
+        )
```

Comment:
Use `"""` instead of `\n`? Or does antimony interpret indentation?

Reply:
Makes sense. No, it doesn't.